release/src/main/scripts/cut_release_branch.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# This script will update apache beam master branch with next release version
+# and cut release branch for current development version.
+
+MASTER_BRANCH=master
+RELEASE=2.6.0
+DEV=2.6.0.dev
+NEXT_VERSION_IN_BASE_BRANCH=2.7.0
+RELEASE_BRANCH=release-${RELEASE}
+GITHUB_REPO_URL=https://github.com/apache/beam.git
+BEAM_ROOT_DIR=beam
+
+# Enforce Release Manager input github credentials every time
+git config --global --unset credential.helper
```

Comment:
I wonder how this will work for those of us that have SSH login set up? Maybe it's no big deal?